alkira/resource_alkira_connector_azure_vnet.go

```diff
@@ -10,58 +10,63 @@ import (
 
 func resourceAlkiraConnectorAzureVnet() *schema.Resource {
 	return &schema.Resource{
+		Description: "Manage Azure Cloud Connector.",
+
 		Create: resourceConnectorAzureVnetCreate,
 		Read:   resourceConnectorAzureVnetRead,
 		Update: resourceConnectorAzureVnetUpdate,
 		Delete: resourceConnectorAzureVnetDelete,
 
 		Schema: map[string]*schema.Schema{
-			"azure_region": &schema.Schema{
+			"azure_region":  {
+				Description: "Region of Azure VNET.",
```

Comment:
To be consistent with AWS
```suggestion
				Description: "Azure Region where VNET resides.",
```